Samples/2a-arm-validation/shell/stderr.txt

```diff
@@ -56,3 +56,61 @@ WARNING (DescriptionMissing/M4000/SDKViolation): 'properties' model/property lac
     - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/d374d03801e97737ddb32e01f20513e7b2bbd9c3/arm-storage/2015-06-15/swagger/storage.json:716:8 ($.definitions.StorageAccountUpdateParameters.properties.properties)
 WARNING (DescriptionMissing/M4000/SDKViolation): 'keyName' model/property lacks 'description' property. Consider adding a 'description' element. Accurate description is essential for maintaining reference documentation.
     - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/d374d03801e97737ddb32e01f20513e7b2bbd9c3/arm-storage/2015-06-15/swagger/storage.json:726:8 ($.definitions.StorageAccountRegenerateKeyParameters.properties.keyName)
+WARNING (NonAppJsonTypeWarning/S2004/SDKViolation): Please make sure that media types other than 'application/json' are supported by your service.
```

Comment:
aaaand this is why we have that CI in place: these messages look duplicate to me, so maybe the validator is run with the *same* mode twice instead of switching between individual and composite? Note that this should also have been caught by an actual test case - this regen diff should always be the last resort of finding out about stuff ;-)